test/sprite-integration.test.ts

```diff
@@ -108,6 +108,22 @@ function filledColorArray(
   return array;
 }
 
+/**
+ * Compare two color arrays and return the proportion of matching pixels.
+ */
+function compareColorArrays(
```

Comment:
Utility function to simplify output on test failure. Without this, each mismatched pixel channel value would be logged to the console (400 lines of output per failure). With this addition, just the proportion of mismatched pixels is logged.